client/components/tooltip/style.scss

```diff
@@ -1,6 +1,10 @@
 $large-tooltip-box-shadow-color: rgba(0, 0, 0, 0.05);
 
 .tooltip.popover {
+
+	// always display the popover on top even inside a modal with a high z index
+	z-index: 100000;
```

Comment:
I remember we used to have some sass functions to manage z-indexes ([similar to Gutenberg](https://github.com/WordPress/gutenberg/blob/308f3ee5ce02192d6129e0764d62ecf5c5ebb6e1/packages/components/src/tooltip/style.scss#L2)).

It makes it easier to manage different z-indexes in different components collectively.

Can you look up if that exists and use it here? Thanks!